tests/core/contracts/test_contract_call_interface.py

```diff
@@ -21,6 +21,7 @@ def deploy(web3, Contract, args=None):
     deploy_receipt = web3.eth.getTransactionReceipt(deploy_txn)
     assert deploy_receipt is not None
     contract = Contract(address=deploy_receipt['contractAddress'])
+    assert len(web3.eth.getCode(contract.address)) > 0
```

Comment:
This my not be accurate anymore because of `HexBytes` but this was always `> 2` to account for the `0x` prefix.

Reply:
Right, this changed with `HexBytes`. I confirmed it while debugging another issue that caused the eth-tester/py-evm deployment to fail (but still return a `contractAddress`!).